test_script.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+testpath="geopyspark/tests/"
+testpattern="*_test.py"
+
+for f in $testpath$testpattern $testpath/**/$testpattern;
```

Comment:
They produce different ones. `$testpath$testpattern` retrieves all of the files in the root directory and `$testpaht/**/$testpattern` gets all of the files from the sub directories.